includes/class-creativecommons.php

```diff
@@ -506,28 +506,48 @@ function select_license_html($location = null, $echo = true)
         // get the previously selected license from this site's options
         // or the plugin's default license
         //$license = get_option('license', $this->plugin_default_license());
-        $license  = $this->get_license($location);
-        // add lang option
-        $lang = (isset($this->locale) && ! empty($this->locale))
-              ? 'lang=' . esc_attr($this->locale) : '';
-
-        $html = '';
-        $html .= "<span id='license-display'></span>";
-        $html .= '<br id="license"><a title="' . __('Choose a Creative Commons license', $this->localization_domain) . '" class="button button-secondary thickbox edit-license" href="https://creativecommons.org/choose/?';
-        $html .= 'partner=CC+WordPress+Plugin&';
-        $html .= $lang;
-        $html .= '&exit_url=' . $this->plugin_url . 'license-return.php?url=[license_url]%26name=[license_name]%26button=[license_button]%26deed=[deed_url]&';
-        $html .= '&KeepThis=true&TB_iframe=true&height=500&width=600">' . __('Change license', $this->localization_domain);
-        $html .=  '</a>';
-
-        $html .= '<input type="hidden" value="'.$license['deed'].'" id="hidden-license-deed" name="license[deed]"/>';
-        $html .= '<input type="hidden" value="'.$license['image'].'" id="hidden-license-image" name="license[image]"/>';
-        $html .= '<input type="hidden" value="'.$license['name'].'" id="hidden-license-name" name="license[name]"/>';
-        if ($echo) {
-            echo $html;
-        } else {
-            return $html;
-        }
+	$license = $this->get_license( $location );
+
+	// Set up link arguments.
+	$link_args = array(
+		'partner'   => 'CC+WordPress+Plugin',
+		'exit_url'  => urlencode( $this->plugin_url ) . 'license-return.php?url=[license_url]%26name=[license_name]%26button=[license_button]%26deed=[deed_url]',
+		'KeepThis'  => 'true',
+		'TB_iframe' => 'true',
+		'height'    => 500,
+		'width'     => 600
+	);
+
+	// Set lang if available.
+	if ( isset( $this->locale ) && ! empty( $this->locale ) ) {
+		$link_args['lang'] = esc_attr( $this->locale );
+	}
+
+	// Link.
+	$link = sprintf( '<a title="%1$s" class="button button-secondary thickbox edit-license" href="%2$s">%3$s</a>',
+		__( 'Choose a Creative Commons license', $this->localization_domain ),
+		add_query_arg( $link_args, 'https://creativecommons.org/choose/' ),
+		__( 'Change license', $this->localization_domain )
+	);
+
+	// HTML markup.
+	$html = <<<CONTENT
```

Comment:
Here we should stick to the wordpress quote standard
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/

Reply:
Feel free to make the change yourself.  We created our own plugin as I mention in #54.